lua/ulx/modules/sh/cfc_arrest.lua

```diff
@@ -0,0 +1,51 @@
+CFCUlxCommands.arrest = CFCUlxCommands.arrest or {}
+local cmd = CFCUlxCommands.arrest
+
+CATEGORY_NAME = "Cleanup"
+
+function cmd.arrest( callingPlayer, targetPlayers )
+    local entities = ents.GetAll()
+    local entCount = 0
+    local entCounts = {}
+    for _, ply in ipairs( targetPlayers ) do
+        entCounts[ply] = 0
+    end
+
+    for _, ent in ipairs( entities ) do
+        local owner = ent.CPPIGetOwner and ent:CPPIGetOwner()
+        if owner and entCounts[owner] then
+            if ent:GetClass() == "gmod_wire_expression2" then
+                ent:PCallHook("destruct")
+                ent:ResetContext()
+                ent:PCallHook("construct")
+                ent:Error( ent.name .. ": Halted by ULX" )
+            end
+
+            if ent:GetClass() == "starfall_processor" then
+                ent:Error( SF.MakeError( ent.name .. ": Halted by ULX", 1, true, true) )
+            end
+
+            local canFreeze = not ( ent:IsWeapon() or ent:GetUnFreezable() or ent:IsPlayer() )
+            local physicsObj = ent:GetPhysicsObject()
+            if IsValid( physicsObj ) and canFreeze then
+                physicsObj:EnableMotion( false )
+                physicsObj:Sleep()
+                entCount = entCount + 1
+                entCounts[owner] = entCounts[owner] + 1
```

Comment:
Can default to 0 here to avoid seeding the table
```suggestion
                entCounts[owner] = ( entCounts[owner] or 0 ) + 1
```